node/builder_miner.go

```diff
@@ -74,6 +74,8 @@ func ConfigStorageMiner(c interface{}) Option {
 	return Options(
 		ConfigCommon(&cfg.Common, enableLibp2pNode),
 
+		Override(CheckFDLimit, modules.CheckFdLimit(100_000)), // recommend at least 100k FD limit to miners
```

Comment:
Also variable/constant? Not critical, but a nice thing to group. Maybe even drop it in build?